src/AuthLDAP.php

```diff
@@ -624,9 +624,12 @@ public function showFormAdvancedConfig()
         echo "</td>";
         echo "</tr>";
 
-        echo "<tr class='tab_bg_1'><td><label for='timeout'>" . __('Timeout') . "</label></td>";
-        echo "<td colspan='3'>";
-
+        echo "<tr class='tab_bg_1'>";
+        echo "<td>" . __('TLS Version') . "</td><td>";
+        Dropdown::showFromArray('tls_version', ['1.0', '1.1', '1.2', '1.3'], ['value' => $this->fields["tls_version"]]);
```

Comment:
```suggestion
        Dropdown::showFromArray('tls_version', ['1.0' => '1.0', '1.1' => '1.1', '1.2' => '1.2', '1.3' => '1.3'], ['value' => $this->fields["tls_version"]]);
```

Reply:
For more readability, I have used a constant.